ts-embed_assets/README.md

```diff
@@ -0,0 +1,61 @@
+# Embedding Assets Sample App
+
+## Run the app locally
+```sh
+ts-node index.ts
+```
+
+Test APIs with:
+```
+$ curl localhost:3000/hello
+hi%
+
+$ curl localhost:3000/static/hello.txt
+Hello Klotho!
+```
+
+## Compile and Deploy with Klotho
+
+run the terminal commands:
+```sh
+# Compile the app
+tsc && klotho . --app $USER-ts-embed-assets -p aws
```

Comment:
1. Missing `npm ci`, right?
2. Do we want/need to require a global install of `tsc`? (The alternative would be to add `typescript` as a dev dependency in the package, and then people can invoke it via `npx tsc`)